src/org/traccar/processing/ComputedAttributesHandler.java

```diff
@@ -95,22 +95,36 @@ protected Position handlePosition(Position position) {
                     result = computeAttribute(attribute, position);
                 } catch (JexlException error) {
                     Log.warning(error);
+                    return position;
                 }
+
                 if (result != null) {
                     try {
                         switch (attribute.getType()) {
                             case "number":
-                                position.getAttributes().put(attribute.getAttribute(), (Number) result);
+                                if (attribute.getAttribute().equals(false) || attribute.getAttribute().equals(true)) {
```

Comment:
Can't we just check type?

Reply:
This was to make sure we are not casting a literal false/true into a number/string

Reply:
It doesn't answer my question. Why don't you just check with `instaceof`?